### PR TITLE
chore: Typo roundup with cSpell

### PR DESCRIPTION
**Pull request by nschonni**
### Why:

Was building out a cSpell dictionary for an eventual CI. The WIP config is over here https://gist.github.com/nschonni/d8655aeb3f5479e79ca29f37b4c658f5, but I figured I'd PR the more straight-forward issues it found

### What's being changed:

- Typos across various content folder documents
- Typos across various GraphQL data elements. This might actually be product bugs being generated from code comments. Each one is separated by commit, so they can be rebased if needed

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging.
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by heiskr — APPROVED** [review]
🎉 Fantastic pull request. `aloglia` has bugged me more than once 😂

File: data/graphql/2.17/schema.docs-enterprise.graphql [hunk 1]
```diff
@@ -2416,7 +2416,7 @@ type Commit implements GitObject & Node & Subscribable & UniformResourceLocatabl
   committedDate: DateTime!
 
   """
-  Check if commited via GitHub web UI.
+  Check if committed via GitHub web UI.
```
Comment (nschonni) [inline data/graphql/2.17/schema.docs-enterprise.graphql:2419]: Just adding a comment here, that these may be generated bugs from the product itself that need to be rebased out of this PR and addressed upstream

Comment (janiceilene) [reply data/graphql/2.17/schema.docs-enterprise.graphql:2419]: Ooh, yeah, let me triage this for the ecosystem team so they can check out whether all of these can be merged 👍

Comment (nschonni) [reply data/graphql/2.17/schema.docs-enterprise.graphql:2419]: No problem, I think these were in the JSON files too, but I didn't touch those

**Review by runleonarun — CHANGES_REQUESTED** [review]
👋 @nschonni Thanks for all of these awesome typo catches! Some of these files are automatically generated and some I just think we should leave because they are comments in scripts that would require an engineering review. Once you remove changes from these directories, this should be good to go:
`tests/`
`scripts/`
`lib/`
`data/graphql/`
`data/glossaries/`

**Comment by nschonni** [comment]
@runleonarun I'll see if I can figure out how to `filter-branch` properly and move those to a separate PR

**Comment by nschonni** [comment]
> 🎉 Fantastic pull request. `aloglia` has bugged me more than once 😂

@heiskr I pulled that change out for the content folks, but I'll resubmit it after this lands. Feel free to copy/submit the fix ahead of that though 😉

**Comment by nschonni** [comment]
@runleonarun I rebased and threw on an extra commit because the heroku deploy stalled on the previous commit

**Review by runleonarun — APPROVED** [review]
This looks great! Thank you for removing those extra changes. 🙇

**Pull request merged**